spec/controllers/wishlists_for_guests_controller_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "rails_helper"
+
+RSpec.describe WishlistsForGuestsController do
+  let(:new_wishlist) { Wishlist.create(name: "Name", description: "Description") }
+  let(:correct_token) { new_wishlist[:token] }
+  let(:incorrect_token) { "000"}
+
+  describe "GET show" do
+    context "when a valid token is provided" do
+      before do
+        get_request(correct_token)
+      end
+
+      it "has a 200 status code" do
+        expect(response.status).to eq(200)
+      end
+    end
+
+    context "when an invalid token is provided" do
+      before do
+        get_request(incorrect_token)
+      end
+
+      it "renders a 404 page" do
+        expect(response).to render_template(:file => "#{Rails.root}/public/404.html")
+      end
+
+      it "has a 200 status code" do
+        expect(response.status).to eq(200)
+      end
+    end
+  end
+
+  private
```

Comment:
@agalkiewicz It's up to us as a team, and I was arogant enough to put Rubocop and Ragnarson's rules for it into our project, so we can maintain a consistent styling. I feel like a douche pointing out formatting details like this, but we **should** all use a consistent coding style. Rubocop is there to enforce this. When you don't use it, then I'm annoying.